core/jvm/src/main/scala/zio/ZIOPlatformSpecificJVM.scala

```diff
@@ -26,4 +26,16 @@ private[zio] trait ZIOPlatformSpecificJVM {
    */
   def fromFutureJava[A](future: => Future[A])(implicit trace: Trace): Task[A] = javaz.fromFutureJava(future)
 
+  /**
+   * Lifts a value of `A`, converting it into an error as an option in the error
+   * channel when its value is `null`, making it easier to interop with Java
+   * code.
+   */
+  final def fromNullable[A](v: => A)(implicit trace: Trace): IO[Option[Nothing], A] =
```

Comment:
Wouldn't it make more sense to use: `A <: AnyRef`?

Reply:
I think that generally makes sense as primitives cannot be null, my worry though is that it might make it annoying to work with generic code as users will need to add the `<: AnyRef` bound.

WDUT, should we side with correctness or UX on this one?

Reply:
I'd say UX 😄

Reply:
Seems like this code should rarely be used in a generic context, but seems fine.